gradle/libs.versions.toml

```diff
@@ -33,10 +35,16 @@ kotest-assertions-core = { module = "io.kotest:kotest-assertions-core", version.
 kotest-assertions-arrow = "io.kotest.extensions:kotest-assertions-arrow:1.2.2"
 kotest-property = { module = "io.kotest:kotest-property", version.ref = "kotest" }
 
+kamel = "com.alialbaali.kamel:kamel-image:0.3.0"
+
 mockk = "io.mockk:mockk:v1.10.2"
 
 kotlin-gradle = { module = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "kotlin" }
 
+chromecast = { module = "su.litvak.chromecast:api-v2", version.ref="chromecast" }
+jackson-annotations = { module ="com.fasterxml.jackson.core:jackson-annotations", version.ref="jackson" }
+jackson-databind = { module ="com.fasterxml.jackson.core:jackson-databind", version.ref="jackson" }
```

Comment:
🥲

Reply:
For some reason we had to manually include `jackson`, it wasn't being pulled from the Chromecast dependency.